tests/queryset/modify.py

```diff
@@ -94,6 +96,30 @@ def test_modify_with_fields(self):
         self.assertEqual(old_doc.to_mongo(), {"_id": 1})
         self.assertDbEqual([{"_id": 0, "value": 0}, {"_id": 1, "value": -1}])
 
+    @needs_mongodb_v26
+    def test_modify_with_push(self):
+        class BlogPost(Document):
+            id = StringField(primary_key=True)
+            tags = ListField(StringField())
+
+        BlogPost.drop_collection()
+
+        BlogPost(id="ABC").save()
+        BlogPost(id="BCD").save()
```

Comment:
If you follow the code I suggested below, you only need one document, hence you can remove this line.